src/cmd/promremotebench/query.go

```diff
@@ -65,12 +78,33 @@ type queryExecutorOptions struct {
 	Debug         bool
 	DebugLength   int
 	Logger        *zap.Logger
+	Scope         tally.Scope
 }
 
 func newQueryExecutor(opts queryExecutorOptions) *queryExecutor {
+	scope := opts.Scope
+	metrics := make(map[string]queryMetrics)
+	errorScope := scope.SubScope("error")
```

Comment:
nit: do you need this subscope? They should already have different names based on the names you pass into the counter creation function

Reply:
Is it better to just prefix the `Counter` names w/ "error-"? Wanted to have error in there.